src/cli/run_command.zig

```diff
@@ -876,26 +876,20 @@ pub const RunCommand = struct {
         if (env == null) {
             this_bundler.env.loadProcess();
 
-            if (this_bundler.env.get("NODE_ENV")) |node_env| {
-                if (strings.eqlComptime(node_env, "production")) {
-                    this_bundler.options.production = true;
-                }
-            }
+            if (this_bundler.env.isProduction())
```

Comment:
so we weren't setting production with `BUN_ENV` before? I think this change should be applied to line 1052 as well